spec/controllers/coronavirus/sub_sections_controller_spec.rb

```diff
@@ -141,8 +144,28 @@
     it "changes the attributes of the subsection" do
       subject
       sub_section.reload
-      expect(sub_section.title).to eq(title)
-      expect(sub_section.content).to eq(content)
+      expect(sub_section.title).to eq("valid title")
+      expect(sub_section.content).to eq("###valid content")
+    end
+
+    context "given invalid content" do
+      let(:params) do
+        {
+          id: sub_section,
+          page_slug: page.slug,
+          sub_section: {
+            title: "valid title",
+            content: "invalid content",
+          },
+        }
+      end
+
+      it "doesn't change the subsection" do
+        subject
+        sub_section.reload
+        expect(sub_section.content).not_to eq("invalid content")
```

Comment:
We can probably use a more concise means to confirm nothing has changed which matches closer to the `it` description.

```suggestion
        expect { subject }.not_to(change { sub_section.reload.attributes })
```

Reply:
nice one, i like it :)